client/src/pages/Listing/ListingItem.tsx

```diff
@@ -49,11 +79,56 @@ export const ListingItem = ({ image, fullName, rating, slogan, description, loca
               <Typography variant="h3" className={fullNameStyle}>
                 {fullName}
               </Typography>
-              <Typography variant="subtitle1">{slogan}</Typography>
+
               <Rating name="read-only" value={rating as number} readOnly />
               <Typography className={descriptionStyle} align="center">
                 {description}
               </Typography>
+              <Box pt={2}>
+                <Divider light />
+                <Chip
```

Comment:
Good thinking adding the available days as a Chip to each individual card. However, it does seem a bit cluttered, I would recommend adding these chips to the Profile Details page instead. Thoughts? :)

Reply:
🙏 
>Please, updated arranged && organized in an orderly way. 😄 
>![image](https://user-images.githubusercontent.com/32603079/139599941-1cffb3b0-572d-416b-b636-33e68b42e629.png)

Reply:
This looks a lot better, thanks for changing it 😆